FIAT/quadrature_schemes.py

```diff
@@ -91,76 +103,313 @@ def _fiat_scheme(ref_el, degree):
     return make_quadrature(ref_el, num_points_per_axis)
 
 
+def _kmv_lump_scheme(ref_el, degree):
+    """Specialized quadrature schemes for P < 6 for KMV simplical elements."""
+
+    sd = ref_el.get_spatial_dimension()
+    # set the unit element
+    if sd == 2:
+        T = UFCTriangle()
+    elif sd == 3:
+        T = UFCTetrahedron()
+
+    if degree == 1:
+        x = ref_el.vertices
+        w = arange(sd + 1, dtype=float64)
+        if sd == 2:
+            w[:] = 1.0 / 6.0
+        elif sd == 3:
+            w[:] = 1.0 / 24.0
+    elif degree == 2:
```

Comment:
`else`'s here and at the end of the block as well.  Also, it easier to sift by `sd` outermost and `degree` inside?